crates/bevy_math/src/direction.rs

```diff
@@ -150,11 +150,39 @@ impl Dir2 {
         Self::new(Vec2::new(x, y))
     }
 
+    /// Creates a new [`Dir2`] from an angle in radians.
```

Comment:
What does this mean?
As a completely naive user I'm not sure what direction this Dir is facing.
I assume it's Dir2::X rotated counterclockwise by the given angle from the impl.

```suggestion
    /// Creates a new [`Dir2`] from an angle in radians by rotating Dir2::X counterclockwise.
```